devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -1413,7 +1413,12 @@ def deleteCode(codeID) {
  * @returns ret: The encapsulated command
  */
 private secure(physicalgraph.zwave.Command cmd) {
-	zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+	// Default to secure encapsulation, unless: zwaveInfo is defined and the device is secure and the command is one we should not encapsulte
+	if (!(zwaveInfo?.zw?.contains("s") && zwaveInfo.sec?.contains(String.format("%02X", cmd.commandClassId)))) {
```

Comment:
Why are we not checking for `state.sec` here and we are else where? Asking more out of curiosity.

Reply:
Older DTHs would set the `sec` variable as a flag when they successfully parsed an encrypted packet. I believe this was before we added the 's' to the end of a secure device's zwaveInfo object.

It's a deprecated practice that we still have to support for legacy reasons.

Reply:
At least in this case, or any case where I don't check it, it is probably because "state.sec" wasn't used anywhere in the code.